syntax = "proto3";

service Restart {
  rpc RestartProcess(RestartRequest) returns (RestartResponse);
  rpc RequestUpdate(UpdateRequest) returns (UpdateResponse);
}

message RestartRequest {}

message RestartResponse {
  string message = 1;
}

message UpdateRequest {}

message UpdateResponse {}
